src/host/screenInfo.cpp

```diff
@@ -113,14 +113,15 @@ SCREEN_INFORMATION::~SCREEN_INFORMATION()
         // Set up viewport
         pScreen->_viewport = Viewport::FromDimensions({ 0, 0 },
                                                       pScreen->_IsInPtyMode() ? coordScreenBufferSize : coordWindowSize);
-        pScreen->UpdateBottom();
 
         // Set up text buffer
         pScreen->_textBuffer = std::make_unique<TextBuffer>(coordScreenBufferSize,
                                                             defaultAttributes,
                                                             uiCursorSize,
                                                             pScreen->_renderTarget);
 
+        pScreen->UpdateBottom();

```

Comment:
This had to move down, because `UpdateBottom` now depends on `_textBuffer` to verify that the bottom hasn't become out of range.